internal/kv/kv.go

```diff
@@ -67,11 +67,18 @@ type MetaKv interface {
 	TxnKV
 	GetPath(key string) string
 	LoadWithPrefix(key string) ([]string, []string, error)
+	CompareVersionAndSwap(key string, version int64, target string, opts ...clientv3.OpOption) (bool, error)
```

Comment:
As an interface, it's better to remove the parameters related to etcd clientv3 implementation 🤔

Reply:
Done.
Not sure why it is there in the first place as there is no usage. It's probably due to legacy reasons.

Reply:
Indeed, it is due to historical reasons. The code iteration is very fast and there are many legacy code that need to be cleaned up.